torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -230,10 +289,12 @@ def sample_inputs_special_polygamma_n(n):
 # to specify if they cannot be auto-generated for some reason. Try to keep these sorted
 # in alphabetical order!
 njt_sample_inputs = {
+    "native_layer_norm": sample_inputs_layer_norm,
```

Comment:
so this PR also adds support for layer norm (as a composite) and softmax?

Reply:
Softmax should be supported already, though this PR updates the logic and adds an OpInfo test.
layer_norm should be already supported as well, and this PR just adds an OpInfo test.